restart-kind-cluster.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+script_path="$(dirname -- "${BASH_SOURCE[0]}")"
+
+main() {
+    echo "Restarting Kind cluster" >&2
+    podman restart konflux-control-plane
+    echo "Updating PID limit" >&2
+    podman update --pids-limit 4096 konflux-control-plane
+    echo "Increasing resource limit" >&2
+    sudo sysctl fs.inotify.max_user_watches=524288
+    sudo sysctl fs.inotify.max_user_instances=512
```

Comment:
This is only needed after the host is restarted. There's no need for rerunning this command if just restarting the container running the cluster.